torch/fx/experimental/proxy_tensor.py

```diff
@@ -115,7 +118,7 @@ def snapshot_fake(val):
     return val.detach()
 
 def extract_val(val):
-    if isinstance(val, FakeTensor):
+    if is_fake_tensor(val):
```

Comment:
this is also a good refactor to extract out

Reply:
will do